rplugin/python3/deoplete/parent.py

```diff
@@ -62,7 +62,6 @@ def merge_results(self, context):
     def on_event(self, context):
         if context['event'] == 'VimLeavePre':
             self._stop_process()
-            self._vim.vars['deoplete#_stopped_processes'] += 1
         self._put('on_event', [context])
```

Comment:
Side note: `_stop_process` should get called at the end likely, since otherwise the `_put` will not do anything.
I assume it is good to propagate the event to childs, and afterwards stop?!

Reply:
@Shougo 
?

Reply:
OK.  It should be.
I will fix it.